react/src/pages/SessionLauncherPage.tsx

```diff
@@ -1410,6 +1420,15 @@ const SessionLauncherPage = () => {
                                                 },
                                               ]}
                                             />
+                                          ) : (
+                                            <Tag
+                                              key={tag.key}
+                                              color={
+                                                isCustomized ? 'cyan' : 'blue'
+                                              }
+                                            >
+                                              {aliasedTag}
```

Comment:
Please add text highlight

Reply:
It's for the `Confirm and Launch` page. Is it needed?